pysat/tests/test_instrument.py

```diff
@@ -51,30 +46,29 @@ def test_basic_instrument_load(self):
         assert (test_date == dt.datetime(2009, 1, 1))
         assert (test_date == self.testInst.date)
 
-    @raises(Exception)
     def test_basic_instrument_bad_keyword(self):
         """Checks for error when instantiating with bad load_rtn keywords"""
-        testInst = pysat.Instrument(platform='pysat', name='testing',
-                                    sat_id='10',
-                                    clean_level='clean',
-                                    unsupported_keyword_yeah=True)
+        with pytest.raises(ValueError):
+            pysat.Instrument(platform='pysat', name='testing', sat_id='10',
+                             clean_level='clean',
+                             unsupported_keyword_yeah=True)
 
-    @raises(Exception)
     def test_basic_instrument_load_yr_no_doy(self):
-        self.testInst.load(2009)
+        with pytest.raises(TypeError):
+            self.testInst.load(2009)
```

Comment:
Change of error type based on running this code manually in ipython (here and elsewhere)